### ISSUE TITLE
Controller not recognized when trying to map inputs for a preset using plugin

### ISSUE THREAD

**Issue by intr-o** [univrsal/input-overlay#398]
When using the overlay preset creator plugin, the controller i am trying to map does not seem to be recognized in any capacity.

*To Reproduce*
Load the latest version of the plugin
load the texture to be used to create the overlay
Select "Add new element"
Select "Gamepad button
select one of the buttons
Select the box next to "keycode" to record the desired input
no worky

*Expected behavior*
I am attempting to create a controller preset for a modded Guitar Hero plastic guitar controller Clone Hero, with the green/orange frets, whammy, etc
The guitar controller im using is a custom Arduino mod that replaces the factory motherboard with well.. an Arduino, and replaces the normal frets with mechanical switches. It also produces easily map-able inputs that windows has little trouble recognizing and using

Additional information:
- Version 5.0.5
 Obs Studio log [2024-06-06 23-20-49.txt](https://github.com/user-attachments/files/15709164/2024-06-06.23-20-49.txt)

**Comment by univrsal** [comment]
Does the program (sdl2.gamepadtest.zip) linked in this issue https://github.com/univrsal/input-overlay/issues/297 recognize the gamepad?

**Comment by intr-o** [comment]
> Does the program (sdl2.gamepadtest.zip) linked in this issue #297 recognize the gamepad?

Sorry i didn't see that program before submitting issue. But yes all of the inputs seem to be recognized by the sdl2 gamepad tester

**Comment by univrsal** [comment]
Is the controller detected when using a tool like https://gamepadtest.com/?

**Comment by intr-o** [comment]
> Is the controller detected when using a tool like https://gamepadtest.com/?

Yes, gamepadtest.com does successfully detect every input from the Arduino modded guitar controller
